### PR TITLE
TaskSelectionTooltips

### PR DESCRIPTION
**Pull request by Shtoner**
This adds tooltips into the task selection page once signed in.

### CODE REVIEW

File: website/src/components/TaskSelection/TaskOption.tsx [hunk 1]
```diff
@@ -12,6 +13,8 @@ export type OptionProps = {
 export const TaskOption = (props: OptionProps) => {
   const { alt, img, title, link } = props;
   return (
+    
+    <Tooltip label={label}>
```
Comment (notmd) [inline website/src/components/TaskSelection/TaskOption.tsx:17]: You didn't deconstruct the label props. Also, please add the type to the `OptionProps`.

File: website/src/components/TaskSelection/TaskSelection.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import { Flex } from "@chakra-ui/react";
+import { Flex, Tooltip } from "@chakra-ui/react";
 import { useColorMode } from "@chakra-ui/react";
 import React from "react";
-
+import styles from '../../styles/Home.module.css'
```
Comment (notmd) [inline website/src/components/TaskSelection/TaskSelection.tsx:4]: this style is not used anywhere, you should remove it.

File: website/src/components/TaskSelection/TaskSelection.tsx [hunk 3]
```diff
@@ -26,46 +26,57 @@ export const TaskSelection = () => {
...
           alt="Reply as Assistant"
           img="/images/logos/logo.svg"
           title="Reply as Assistant"
           link="/create/assistant_reply"
+          label='Respond to the user as the AI'
+
         />
       </TaskOptions>
-      <TaskOptions key="evaluate" title="Evaluate">
+      <TaskOptions key="evaluate" data-size="Evaluate">
```
Comment (notmd) [inline website/src/components/TaskSelection/TaskSelection.tsx:49]: Accident?

Comment (Shtoner) [reply website/src/components/TaskSelection/TaskSelection.tsx:49]: Indeed. I saw you said this is currently not being used but I'll leave it up to you if it should be committed for possible future cases if we go back to these components or if they will be deleted all together.

**Comment by fozziethebeat** [comment]
> I believe the TaskSelection component is not currently being used. With PR #359 we have a dashboard page that already has self-explanatory entries. Here is an example: Example

This is correct.  This change now needs to go into `website/src/components/Widgets/TaskOption.tsx`

**Pull request closed without merging**